src/tests/ftest/erasurecode/restart.py

```diff
@@ -47,6 +47,7 @@ def execution(self, agg_check=None):
         self.ior_write_dataset(operation="Auto_Write", percent=self.percent)
         free_space_after_ior = self.pool.get_total_free_space(refresh=True)
         self.log.info("pool free space after IOR write: %s", free_space_after_ior)
+        self.assertTrue(free_space_after_ior < initial_free_space, "IOR run was not successful.")
```

Comment:
1. We should use `assertLess` so the values show up in the error message.
2. We should make the error message more accurate, since IOR was successful - it's the free space check that isn't successful
Maybe:
```suggestion
        self.assertLess(free_space_after_ior, initial_free_space, "Free space not reclaimed after IOR write")
```

Reply:
Waiting for Phil's comment, if there is more changes.